projects/user_publication/utils.py

```diff
@@ -73,6 +99,27 @@ def guess_project_for_publication(authors, pub_year):
     return counter.most_common(1)[0][0]
 
 
+def report_publications(pubs, display=True):
+    line_format = "{0:18} : {1}\n"
+    for pub in pubs:
+        pd = pub.__dict__
+        report = [line_format.format(ck, pd.get(ck)) for ck in PUBLICATION_REPORT_KEYS]
```

Comment:
For this, it might be a good idea to implement `Publication.__repr__` so we could reuse it for logging in other places if desired.